doc/readme.md

```diff
@@ -315,6 +315,7 @@ cordova {plugin | plugins} [
 |       |--link                   | When installing from a local path, creates a symbolic link instead of copying files. The extent to which files are linked varies by platform. Useful for plugin development.
 |       |--save                   | Save the `<plugin-spec>` as part of the `plugin` element  into `config.xml`.
 |       |--browserify             | Compile plugin JS at build time using browserify instead of runtime.
+|       |--force                  | Forces copying source files from the plugin even if the same file already exists in the target directory.
```

Comment:
Yup, I rebased and changed readme.md as you requested.
Please merge all 5 pull requests in CLI, LIB, ANDROID, IOS, WINDOWS.
Thanks.